recipes/gnu-time/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "gnu-time" %}
```

Comment:
The binary name is to be considered too. I chose to use `gtime` instead of `time` as `time` is a bash builtin. The difference is that `gnu-time` has more options (needed by something I have in the pipeline at bioconda).